src/main/java/racingcar/view/OutputView.java

```diff
@@ -1,24 +1,38 @@
 package racingcar.view;
 
 import racingcar.model.Cars;
+import racingcar.view.dto.RacingResult;
 
 public class OutputView {
     private static final String WINNER_MESSAGE = "가 최종 우승했습니다.";
+    private static final String NO_WINNER_RESULT_MESSAGE = "우승 결과가 없습니다.";
 
     private OutputView() throws InstantiationException {
         throw new InstantiationException();
     }
 
-    public static void outputStatus(Cars cars) {
-        cars.getStatus().forEach(System.out::println);
-        outputEmptyLine();
-    }
-
     private static void outputEmptyLine() {
         System.out.println();
     }
 
-    public static void outputWinners(Cars winnerCars) {
+    public static void outputWinners(RacingResult racingResult) {
+        if (racingResult == null) {
+            System.out.println(NO_WINNER_RESULT_MESSAGE);
+            return;
+        }
+
+        Cars cars = getCarsFrom(racingResult);
+        Cars winnerCars = cars.findWinners();
```

Comment:
`RacingResult`에서 `Cars`를 꺼내는 대신, `RacingResult`에게 메시지를 보내서 우승자를 구해보면 어떨까요?
```suggestion
        Cars winnerCars = racingResult.getWinners();
```